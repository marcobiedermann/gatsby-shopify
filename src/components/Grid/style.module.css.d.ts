declare const styles: {
  readonly grid: string;
  readonly gridFull: string;
};
export = styles;
