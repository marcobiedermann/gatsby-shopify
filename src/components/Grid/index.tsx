import React, { FC } from 'react';
import styles from './style.module.css';

const Grid: FC = (props) => {
  return <div className={styles.grid} {...props} />;
};

export default Grid;
