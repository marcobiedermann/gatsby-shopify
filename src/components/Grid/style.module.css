.grid {
  margin-left: auto;
  margin-right: auto;
  max-width: 1440px;
  width: 90%;
}
