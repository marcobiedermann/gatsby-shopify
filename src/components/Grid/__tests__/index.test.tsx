import { render } from '@testing-library/react';
import React from 'react';
import Grid from '..';

describe('grid component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Grid>Grid</Grid>);

    expect(asFragment()).toMatchSnapshot();
  });
});
