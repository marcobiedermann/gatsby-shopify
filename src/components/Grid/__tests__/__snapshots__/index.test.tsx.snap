// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grid component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="grid"
  >
    Grid
  </div>
</DocumentFragment>
`;
